integration_tests/src/client.rs

```diff
@@ -114,7 +114,7 @@ impl InteropClient {
         } else {
             InteropClient {
                 name: "us-west2-docker.pkg.dev/divviup-artifacts-public/divviup-ts/divviup_ts_interop_client".to_string(),
-                tag: "dap-draft-02@sha256:dc4c111adce5719fef9d231f79aecb842c3f626d51363b04355e987d6a40aadc".to_string(),
+                tag: "dap-draft-03@sha256:b03102a4e8eb9e1abf879940aa89d6a04e4a8229a215c715a4c92a0aec4b3be7".to_string(),
```

Comment:
This image also appears in `.github/workflows/ci-build.yml`, and should be updated there. (We specify this default both here, for developer use, and in the workflow, because we wanted to be able to override the image when using a `workflow_dispatch` trigger)